pixeltable/tests/conftest.py

```diff
@@ -74,6 +74,13 @@ def test_tbl_exprs(test_tbl: catalog.Table, test_stored_fn: pt.Function) -> List
         (t.c2 > 5) | (t.c1 == 'test'),
         t.c7['*'].f5 >> [R[3], R[2], R[1], R[0]],
         t.c8[0, 1:],
+        t.c1.astype(StringType()),
+        t.c2.astype(StringType()),
+        t.c3.astype(StringType()),
+        t.c4.astype(StringType()),
+        t.c5.astype(StringType()),
+        t.c6.astype(StringType()),
+        t.c1.astype(JsonType()),
```

Comment:
Also add an apply()

Reply:
Oops, good catch. I completely forgot to update this when the semantics of `astype` changed.

I'm surprised nothing broke here. What is this test artifact actually being used for?